pymeasure/instruments/newport/esp300.py

```diff
@@ -175,7 +175,7 @@ class Axis(object):
         "MD?",
         """ Returns a boolean that is True if the motion is finished.
         """,
-        cast=bool
+        get_process=lambda x: bool(x)
```

Comment:
Did you merge in the latest `master` branch?

Reply:
Oh, I thought I had. Since this seems to be the only thing missing from `master`, I've changed it back by hand.